index.bs

```diff
@@ -746,6 +746,22 @@ following values:
     the message receiver chooses the connection-id, it may keep the ID unique
     across connections, thus making message demuxing/routing easier).
 
+: connection-count
+:: The new total number of connections open to the presentation that received
+    the incoming connection request.
+
+If the `presentation-connection-open-response` message indicates success, the
+receiver should also send a `presentation-change-event` to all other endpoints
+that have an active presentation connection to that presentation with the
+values:
```

Comment:
Shouldn't we also send one when a presentation is closed (when the number goes down?)

Reply:
Makes sense, for symmetry. So we'd want to add the count to the `presentation-termination-response` and `presentation-termination-event` messages.

Reply:
Should the name be `presentation-connection-change-event` for consistency with the other `presentation-connection-xxx` messages we have?

Reply:
@pthatcherg Good point.  Added connection-count to presentation-connection-close-event, and how have the receiver send a change-event if it closes a connection.

@chrisn I went with presentation-change-event since we are announcing a change to a property of the presentation itself (the number of open connections), instead of a change to an individual connection.

Reply:
Also, @chrisn, terminating a presentation must close all of its connections.  So I don't think we need to add a count to the termination messages.

Reply:
Ah, I was thinking of connection termination rather than presentation termination. So, you're right, the change looks good. Thanks!